src/client/failure/policy.ts

```diff
@@ -1,3 +1,6 @@
+import { HttpResponse } from '@superfaceai/one-sdk';
```

Comment:
fix this pls and do something about this, this happens to you a lot for some reason :suspect:

Reply:
Yep, my vscode is borked when it comes to imports, already tried reinstall. Good thing is that lint in CI catches this stupid stuff.